docs/samples/browser-plugin-meetings/app.js

```diff
@@ -697,6 +705,24 @@ function getMediaSettings() {
   return settings;
 }
 
+// in case add/update media function fails then we need to restore the previous media setting toggles
+// reset will make all media toggles as default checked(in case of addMedia() failure as it is the initial case)
+function setLastMediaSettingToggles(reset, previousMediaSettings) {
```

Comment:
to me it feels like the code would be more readable if there were 2 separate functions:
1. resetMediaSettingsToDefaults() that does what calling `setLastMediaSettingToggles(true)` does
2. revertMediaSettings(changedMediaSettings) that does what calling `setLastMediaSettingToggles(false, changedSettings)` does

Reply:
Done